src/content/en/updates/2017/12/chrome-64-deprecations.md

```diff
@@ -0,0 +1,70 @@
+{# wf_featured_snippet: A round up of the deprecations and removals in Chrome 64 to help you plan. In this version, performance API changes, removal of support for multiple shadow roots, and removal of a webkit API. #}
+
+# Deprecations and removals in Chrome 64 {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+In nearly every version of Chrome, we see a significant number of updates and
+improvements to the product, its performance, and also capabilities of the Web
+Platform. This article describes some of the deprecations and removals in Chrome
+64, which is in beta as of December 14. Visit the
+[deprecations page](/web/updates/tags/deprecations) for more deprecations and
+removals from this and previous versions of Chrome. This list is subject to
+change at any time.
+
+## Remove chrome.loadTimes()
+
+The `chrome.loadTimes()` method is a non-standard API that enables collecting
+loading-related metrics in real-world operations. A [Resource Timing Level
+1](https://www.w3.org/TR/resource-timing-1/) API has existed for several years
+while the current version of it is a W3C Candidate Recommendation. Chrome itself
+has had some version of an implementation [since Chrome
+25](https://www.chromestatus.com/features/5796350423728128). For these reasons,
+the `chrome.loadTimes()` method is removed as of Chrome 64.
+
+Developers needing an alternative should look specifically at the
+[`PerformanceResourceTiming` interface](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming).
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/QqvFGFgoTyI/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/features/5637885046816768) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=621512)
+
+## Remove support for multiple shadow roots
+
+Shadow DOM version 1 required multiple shadow roots. At a [standars meeting in
+April 2015](https://www.w3.org/wiki/Webapps/WebComponentsApril2015Meeting) it
+was decided that this feature should not be part of version 2. Supported was
+deprecated shortly thereafter in Chrome 45. In Chrome 64 support is now removed.
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/JjtmigNE28M/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/features/4668884095336448) &#124;
+[Chromium Bug](https://code.google.com/p/chromium/issues/detail?id=489947)
+
+## Remove getMatchedCSSRules()
+
+The `getMatchedCSSRules()` method is a non-standard, webkit-only API that
+retrieves a list of style rules applied to a particular element. This has been
+deprecated since 2014. It's now being removed because it's not on a standards
+track.
+
+Since there is currently no standards-based alternative, developers would need
+to create their own. There is at least [one example on
+StackOverflow](https://stackoverflow.com/questions/2952667/find-all-css-rules-
+that-apply-to-an-element).
```

Comment:
Maybe we can add [this polyfill](https://gist.github.com/darrnshn/addeabe2575177342cc6242e20ecadbd) as another example? It's not an official polyfill and it's not actively maintained, but serves as a good example.

Reply:
Will this stay up if I link to it?

Reply:
I'm not going to delete it or anything, it's just not maintained, so I guess it'll stay up in the foreseeable future?

Reply:
I'm using that link for now. But I want to move this to a Google location to ensure it stays up for a while.

Reply:
Hmm, would moving it to a Google location make it "official" and be obligated to maintain it? We're trying to avoid maintaining it because unlike an actual polyfill, this probably won't be replaced by a standards-based method in the near future.

Maybe it's best to not link to the polyfill then. The stackoverflow answer is already pretty good and people can still fill the polyfill from the intent to remove bug or by googling. @wilddamon, WDYT?

Reply:
Ok instead of linking to the polyfill, how about we just mention that that intent to remove has more alternatives?